build.gradle

```diff
@@ -0,0 +1,93 @@
+repositories {
+    mavenCentral()
+}
+
+configurations {
+    toCopy
+}
+
+dependencies {
+    toCopy 'io.swagger.codegen.v3:swagger-codegen-cli:3.0.47'
+}
+
+task downloadSwaggerCodegen(type: Copy) {
+    from configurations.toCopy
+    into 'lib'
+    rename {
+        String fileName -> "swagger-codegen-cli.jar"
+    }
+}
+
+task createLanguageModule() {
+    dependsOn downloadSwaggerCodegen
+    description = 'Generate a Swagger Codegen Module for a new language'
+
+    doFirst {
+        if (!project.hasProperty('language')) {
+            throw new GradleException("The 'language' property is required. Please use -Planguage=<yourLanguage>.")
+        }
+    }
+    doLast {
+        def language = project.language
+        exec {
+            commandLine 'java', '-jar', 'lib/swagger-codegen-cli.jar', 'meta',
+                '-o', "templates/${language}",
```

Comment:
I think we're probably going to want a different hierarchy.

Specifically, I think we might want to have something like:

```
- generators
| - { language }
| | - library
| | | - ... codegen module files go here
| | config.yml.template
```

The config template would be contain an `envsubst` configuration template that can be passed to swagger-codegen (templates in swagger-codegen can accept paramaters that get used by the generator.  For instance, we might want to make a configurable version number that gets included in the generated `package.json`)